app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in maths.</p>
+<p>Maths teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a maths teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge. </p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+<p><a href="/non-uk-teachers">Non-UK citizens can find out more about training to teach in England</a>.</p>
+
+<h2>Fund your teacher training</h2>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee maths teachers</a>. </p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get <a href="/funding-and-support">extra funding support</a> if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
```

Comment:
@caitlin-reidy there was a query from Erika about using unlikely. It might be clearer to say something similar to what's on the international page? Most non-UK citizens will not be eligible for funding

Reply:
@MaxineCl im waiting to hear back from laura about this